src/patternfly/demos/AboutModal/examples/about-modal-example.hbs

```diff
@@ -1,7 +1,7 @@
 {{#> about-modal}}
   {{#> backdrop}}
     {{#> bullseye}}
-      {{#> about-modal-box about-modal-box--attribute='aria-labelledby="about-modal-title"'}}
+      {{#> about-modal-box title--ID="about-modal-title" about-modal-box--attribute='aria-labelledby="about-modal-title"'}}
```

Comment:
This isn't the correct update for this issue. This line should go back to what it was:
`{{#> about-modal-box about-modal-box--attribute='aria-labelledby="about-modal-title"'}}`

Reply:
This is the [line for the component example](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/AboutModalBox/examples/about-modal-box-example.hbs#L12) that that shows what we need in the demo example:
`{{#> title titleType="h1" title--modifier="pf-m-4xl" title--attribute='id="about-modal-title"'}}`

In the demo example, the id is defined as:
`title--ID="about-modal-title"`
But instead it needs to be defined as:
`title--attribute='id="about-modal-title"'`